contracts/amm/AmmDataProvider.sol

```diff
@@ -405,7 +407,7 @@ contract AmmDataProvider is IAmmDataProvider {
     /// representing the price as a fraction of 1 collateral token unit
     /// @dev This function assumes that it will only be called on an OPEN Series; if the
     /// Series is EXPIRED, then the expirationDate - block.timestamp will throw an underflow error
-    function getPriceForExpiredSeries(uint64 seriesId, uint256 annualVolatility)
+    function getPriceForSeries(uint64 seriesId, uint256 annualVolatility)
```

Comment:
Was this just a mis-named function before?

Reply:
Yes it was getting price of Series not expired series.

Reply:
OK cool...